tools/system_libs.py

```diff
@@ -232,6 +232,12 @@ def create_libc(libname):
         'faccessat.c',
     ]
 
+    # any dirent file that digs into DIR structure
```

Comment:
Can you say why in the comment.  Something like: ".. directory handling in implemented JS"?

Reply:
Sure